Podfile

```diff
@@ -4,15 +4,18 @@ platform :ios, '8.0'
 
 inhibit_all_warnings!
 
-pod 'MTDates'
-pod 'AFNetworking'
-pod 'MTCardLayout'
-pod 'MBProgressHUD'
-pod 'IHKeyboardAvoiding'
+target 'SelfConference' do
+    pod 'MTDates'
+    pod 'AFNetworking'
+    pod 'MTCardLayout'
+    pod 'MBProgressHUD'
+    pod 'IHKeyboardAvoiding'
```

Comment:
🧁Civilized persons alphabetize their pods

Reply:
I'll revisit that as a tech debt issue 😆